application/src/main/java/org/thingsboard/server/service/queue/processing/AbstractConsumerService.java

```diff
@@ -136,6 +136,15 @@ protected void launchNotificationsConsumer() {
     @PreDestroy
     public void destroy() {
         stopped = true;
+
```

Comment:
I have intentionally removed this. Kafka consumers will throw exceptions here (ConcurrentModificationException). Please refactor Kafka consumers to make sure they are unsubscribing in main consumer thread (however, if the executor is shut down - this is meaningless).